tests/base-notebook/test_packages.py

```diff
@@ -106,53 +107,44 @@ def r_package_predicate(package: str) -> bool:
     return not excluded_package_predicate(package) and package.startswith("r-")
 
 
-def _check_import_package(
-    package_helper: CondaPackageHelper, command: list[str]
-) -> int:
+def _check_import_package(package_helper: CondaPackageHelper, command: list[str]):
     """Generic function executing a command"""
     LOGGER.debug(f"Trying to import a package with [{command}] ...")
-    rc = package_helper.running_container.exec_run(command)
-    return rc.exit_code  # type: ignore
+    exec_result = package_helper.running_container.exec_run(command)
+    assert (
+        exec_result.exit_code == 0
+    ), f"Import package failed, output: {exec_result.output}"
```

Comment:
This way we do not lose the output of the test and using exceptions to show the errors is more pythonic.